### PR TITLE
Graduate Google Workspace components out of `/beta` folder

### PR DESCRIPTION
**Pull request by ag-adampike**
The `/beta` folder in this repository includes deployment examples and SCIM bridge components that require testing and feedback and may change before being implemented as (or added as part of) one of our vetted deployment examples.

For customers integrating with Google Workspace, additional configuration is required for all deployment examples, and in the case of the Terraform deployment, an additional submodule.

As this configuration is now mature and well-vetted, this PR:

- removes the settings file template (`workspace-settings.json`) from the `/beta` folder and adds it to each of our deployment example folders
- moves the Terraform module for the Google Workspace configuration into the Terraform deployment example folder
- obviates the Workspace settings file for the Terraform deployment in favour of a supplied value
- updates the READMEs for each deployment example accordingly

### CODE REVIEW

File: aws-ecsfargate-terraform/README.md [hunk 1]
```diff
@@ -34,50 +34,23 @@ cp terraform.tfvars.template terraform.tfvars
 ```
 
 <details>
-  <summary>Optional: If you use Google Workspace</summary>
+  <summary>For customers integrating with Google Workspace:</summary>
```
Comment (Penderworth) [inline aws-ecsfargate-terraform/README.md:37]:
```suggestion
  <summary>Integrate with Google Workspace</summary>
```

File: aws-ecsfargate-terraform/README.md [hunk 2]
```diff
@@ -34,50 +34,23 @@ cp terraform.tfvars.template terraform.tfvars
...
-Copy your `workspace-credentials.json` file to this Terraform code directory:
-
-```bash
-cp <path>/workspace-credentials.json ./workspace-credentials.json
-```
-
-### Enable Google Workspace configuration
-
-Uncommment this line in `terraform.tfvars`:
+1. Create the service account, key, and API client (see [Connect Google Workspace to 1Password SCIM Bridge](https://support.1password.com/scim-google-workspace/#step-1-create-a-google-service-account-key-and-api-client) on our support site for instructions.)
```
Comment (Penderworth) [inline aws-ecsfargate-terraform/README.md:41]:
```suggestion
1. [Create a service account, key, and API client.](https://support.1password.com/scim-google-workspace/#step-1-create-a-google-service-account-key-and-api-client)
```

File: aws-ecsfargate-terraform/README.md [hunk 3]
```diff
@@ -34,50 +34,23 @@ cp terraform.tfvars.template terraform.tfvars
...
+2. Save the Google Workspace service account key to the working directory as `workspace-credentials.json` (or rename it to this after saving).
+3. Uncomment this line in `terraform.tfvars` and fill in the email address for the Google Workspace administrator associated with the above credentials:
```
Comment (Penderworth) [inline aws-ecsfargate-terraform/README.md:43]:
```suggestion
2. Save the Google Workspace service account key to the working directory.
3. Make sure the service account key is named `workspace-credentials.json`.
4. Uncomment the following line in `terraform.tfvars` and and enter the email address for the Google Workspace administrator associated with the credentials you just created.
```

File: aws-ecsfargate-terraform/modules/google-workspace/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+This is a Terraform module to manage the configuration required to integrate 1Password with Google Workspace for automated provisioning. The module is not necessary for customers integrating with any other identity provider, and is not used unless explicitly enabled by the Terraform deployment example module for 1Password SCIM Bridge.
+
+This module:
+
+- Creates AWS secrets to store the Google Workspace config
+- Creates an IAM policy to read the secrets and attaches it to the role passed into the module
+- Outputs a list of `secrets` objects to append to the SCIM bridge container definition
+
+Detailed instructions on how to enable and use this module are avaible in the [`README.md`](../README.md#step-1-configure-the-deployment) document in the Terraform deployment example folder.
```
Comment (plttn) [inline aws-ecsfargate-terraform/modules/google-workspace/README.md:11]: `s/avaible/available`

File: aws-ecsfargate-terraform/terraform.tfvars.template [hunk 5]
```diff
@@ -18,5 +18,5 @@ tags = {
 # Uncomment the below line if you are *not* using Route 53
 #using_route53 = false
 
-# Uncomment the below line to enable Google Workspace configuration for 1Password SCIM bridge
-#using_google_workspace = true
+# Uncomment the below line and specify the approriate email address to enable connecting to Google Workspace
```
Comment (plttn) [inline aws-ecsfargate-terraform/terraform.tfvars.template:21]: `s/approriate/appropriate`

File: aws-ecsfargate-terraform/modules/google-workspace/README.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# 1Password SCIM Bridge Google Workspace module
+
+This is a Terraform module to manage the configuration required to integrate 1Password with Google Workspace for automated provisioning. The module is not necessary for customers integrating with any other identity provider, and is not used unless explicitly enabled by the Terraform deployment example module for 1Password SCIM Bridge.
```
Comment (Penderworth) [inline aws-ecsfargate-terraform/modules/google-workspace/README.md:3]:
```suggestion
This is a Terraform module to manage the configuration required to integrate 1Password with Google Workspace for automated provisioning. If you want to integrate with a different identity provider, this module isn't necessary and won't be used.
```

File: aws-ecsfargate-terraform/modules/google-workspace/README.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+This module:
+
+- Creates AWS secrets to store the Google Workspace config
+- Creates an IAM policy to read the secrets and attaches it to the role passed into the module
+- Outputs a list of `secrets` objects to append to the SCIM bridge container definition
```
Comment (Penderworth) [inline aws-ecsfargate-terraform/modules/google-workspace/README.md:9]:
```suggestion
- Creates AWS secrets to store the Google Workspace configuration.
- Creates an IAM policy to read the secrets and attaches it to the role passed into the module.
- Outputs a list of `secrets` objects to append to the SCIM bridge container definition.
```

File: aws-ecsfargate-terraform/modules/google-workspace/README.md (hunk 4 above)
Comment (Penderworth) [inline aws-ecsfargate-terraform/modules/google-workspace/README.md:11]:
```suggestion
Learn how to use this module in the Terraform deployment example folder's [`README.md`](../README.md#step-1-configure-the-deployment) document.
```

File: beta/docker/README.md [hunk 8]
```diff
@@ -127,14 +127,8 @@ Additional configuration and credentials are required to integrate to integrate
...
-1. Save the Google Workspace service account key to this working directory (or rename after saving) as `workspace-credentials.json`.
-2. Copy the [Google Workspace settings template file](/beta/workspace-settings.json) in this repository to the working directory:
-
-    ```sh
-    cp ../workspace-settings.json ./workspace-settings.json
-    ```
-
-    Edit the file. Replace the values for each key:
+1. Save the Google Workspace service account key file to this working directory (or rename after saving) as `workspace-credentials.json`.
+2. Open the Google Workspace settings template file ([`workspace-settings.json`](./workspace-settings.json)) in this directory in a text editor. Replace the values for each key:
```
Comment (Penderworth) [inline beta/docker/README.md:131]: Similar to my suggestion in the other article:

```suggestion
1. Save the Google Workspace service account key to the working directory.
2. Make sure the service account key is named `workspace-credentials.json`.
3. Open the  [`workspace-settings.json`](./workspace-settings.json) file in a text editor and replace the values for each key:
```

**Review by plttn — DISMISSED** [review]
Overall looks good to me and makes things much clearer. 

minor typos flagged 🙂

File: kubernetes/README.md [hunk 9]
```diff
@@ -46,16 +46,21 @@ The following steps only apply if you use Google Workspace as your identity prov
...
-	```bash
-	kubectl create secret generic workspace-credentials --from-file=workspace-credentials.json=/path/to/<keyfile>.json
-	```
-3. Edit the file located at `scim-examples/beta/workspace-settings.json` and fill in correct values for:
+
+   ```bash
+   kubectl create secret generic workspace-credentials \
+       --from-file=workspace-credentials.json=/path/to/<keyfile>.json
+   ```
+
```
Comment (Penderworth) [inline kubernetes/README.md:54]: Trimming some extra space so the list renders properly.
```suggestion
   ```bash
   kubectl create secret generic workspace-credentials \
       --from-file=workspace-credentials.json=/path/to/<keyfile>.json
   ```
```

**Review by Penderworth — CHANGES_REQUESTED** [review]
Looking good! I left a few suggestions and am happy to take another look once they're considered/addressed.

**Review by Penderworth — APPROVED** [review]

**Pull request merged**